### PR TITLE
[Document] Improve documentation with a straight to the point example

### PR DESCRIPTION
**Pull request by ryuukk**
Improves documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -293,10 +293,18 @@ depending on the shell you use, then restart your console.
 
 # Examples
 
-See the [documentation](docs/README.md) for specific walkthroughs,
-including [installing and using a package](docs/examples/installing-and-using-packages.md),
-[adding a new package from a zipfile](docs/examples/packaging-zipfiles.md),
-and [adding a new package from a GitHub repo](docs/examples/packaging-github-repos.md).
+```
+vcpkg install mongoose[ssl]:x64-windows-static
```
Comment (Neumann-A) [inline README.md:297]:
```suggestion
vcpkg install mongoose[ssl]:x64-windows-static --host-triplet=x64-windows
```
+ add description

File: README.md [hunk 2]
```diff
@@ -293,10 +293,18 @@ depending on the shell you use, then restart your console.
...
+             |        |    | 
+             package  |    target
+                      |
+                      feature
```
Comment (dg0yt) [inline README.md:301]:
~~~
vcpkg install mongoose[core,ssl]:x64-windows-static
                       |
                       feature list

Comment (vicroms) [inline README.md:301]:
```suggestion
vcpkg install mongoose[ssl]:x64-windows-static
              |       |    | 
              package |    target triplet
                      |
                      feature list
```

fix off-by-one in format

**Review by vicroms — CHANGES_REQUESTED** [review]

**Comment by Cheney-W** [comment]
@ryuukk  Can you address the review suggestions? Thanks!

**Comment by Cheney-W** [comment]
Pinging @ryuukk for response. Is work still being done for this PR?

**Comment by vicroms** [comment]
The only problem I have with this type of formatting is that it doesn't work well with screen readers. I'd rather we use a proper image with alt-text or do a better job of explaining this topic textually.

File: README.md (hunk 1 above)
Comment (BillyONeal) [reply README.md:297]: I don't expect most people to need to pass `--host-triplet` so I think it's better to not include it in the first front-line example.

**Review by ras0219-msft — COMMENTED** [review]
A few concerns:
- As @vicroms mentioned above, screen readers interact poorly with ascii art, so at a minimum we should look at leveraging html accessibility practices. https://stackoverflow.com/questions/22676072/how-do-i-hide-ascii-art-from-screen-readers-and-other-at-using-aria-html5 looks like a good start.
- This presents classic mode as the primary and only example, however many users are best served by manifest mode. If we're going to present an inline example like this, I think it should be our "best foot forward" -- manifest mode.

**Comment by BillyONeal** [comment]
Dropping vcpkg-team-review because all the folks who should review already have.

**Comment by Cheney-W** [comment]
Pinging @ryuukk for response. Is work still being done for this PR?

**Comment by Cheney-W** [comment]
Closing this PR since it seems that no progress is being made. Please reopen if work is still being done.

**Pull request closed without merging**